gettsim/parameters/arbeitsl_geld_2.yaml

```
@@ -221,6 +221,14 @@ regelsatz:
     5: 309
     6: 283
     reference: G. v. 09.12.2020 BGBl. I S. 2855.
+  2022-01-01:
+    1: 449
+    2: 404
+    3: 360
+    4: 376
+    5: 311
+    6: 285
+    reference: V. v. 13.10.2021 BGBl. I S. 4674.
```

most recent:

```suggestion
    reference: V. v. 23.09.2021 BGBl. I S. 4389.
```

see https://offenegesetze.de/veroeffentlichung/bgbl1/2021/73#page=26

Footnote:
> * Diese Verordnung ersetzt die gleichnamige Verordnung vom 23. September 2021 (BGBl. I S. 4389)